openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -127,6 +127,9 @@ Both the Wallet and the Credential Issuer:
 
 Both Wallet initiated and Issuer initiated issuance are supported.
 
+The Wallet MUST support batch issuance. For each Credential Configuration, the Issuer MUST indicate whether batch issuance is supported by including or omitting the `batch_credential_issuance` metadata parameter. The Issuer’s decision may be influenced by various factors, including, but not limited to, trust framework requirements, regulatory constraints, applicable laws or internal policies.
```

Comment:
```suggestion
The Wallet MUST support batch issuance. The Issuer MUST indicate whether batch issuance is supported by including or omitting the `batch_credential_issuance` metadata parameter. The Issuer’s decision may be influenced by various factors, including, but not limited to, trust framework requirements, regulatory constraints, applicable laws or internal policies.
```
Batch issuance is a server-wide setting, so cannot be done per credential configuration.